src/print_util.h

```diff
@@ -0,0 +1,24 @@
+#pragma once
```

Comment:
Use
```
#ifndef PRINT_UTIL_H
#define PRINT_UTIL_H
...
#endif

Reply:
Why? 
As far as I know "#pargma once" has good support for a bout 5+ years.

Reply:
True, it's really just a style thing, trying to keep things consistent.